CHANGELOG.md

```diff
@@ -10,8 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
+- Changed default `chunk_size` of various `download` functions from None to 10MB. This should improve download speed of large files.
```

Comment:
I'm not sure it will improve the speed, I would just say that it improves handling of large downloads